SECURITY-INSIGHTS.yml

```diff
@@ -32,7 +32,7 @@ security-artifacts:
   self-assessment:
     self-assessment-created: true
     evidence-url:
-    - https://github.com/jaegertracing/jaeger/blob/main/SELF-ASSESMENT.md
+    - https://github.com/cncf/tag-security/blob/main/assessments/projects/jaeger/self-assessment.md
```

Comment:
then what do we do with https://github.com/jaegertracing/jaeger/blob/main/SELF-ASSESMENT.md ?

Reply:
Good catch, just removed it.